src/config/loadGitHubApps.ts

```diff
@@ -0,0 +1,14 @@
+import { AppAuthStrategyOptions } from '@api/github/appAuthStrategyOptions';
+
+export function loadGitHubApps(env) {
```

Comment:
does this function need to pass in environment variables as `env`? Can we just use `process.env` here?

Reply:
It makes testing much easier.

Reply:
In that case, can we just pass in `GH_APP_IDENTIFIER` and `GH_APP_SECRET_KEY` instead? feels weird to me to pass all the env variables just to use two of them

Reply:
Filtering out unused envvars is part of the logic we want to test, no? Does da690d21b0e35db187ab60f147291a4d9e40a834 help? It changes the function name to `loadGitHubAppsFromEnvironment` and adds a few more tests.

Reply:
Also keep in mind that the next PR is going to add multi-org support, for which we're going to need to iterate over the whole process.env to pull out the vars we want.